src/bin/cross-util.rs

```diff
@@ -6,8 +6,24 @@ use cross::{docker, rustc::Toolchain};
 
 mod commands;
 
+const APP_NAME: &str = "cross-util";
+static mut VERSION: String = String::new();
+
+fn version() -> &'static str {
+    // safe since single-threaded execution
+    unsafe {
```

Comment:
I've used to avoid the `once_cell` dependency in the main crate. I can change it if desired.

Reply:
yes please, let's use once_cell, soon it's in stable also so that's nice :)

however, is it really needed? Think it would be fine to do `name = "cross-util", version = format!("{}{}", env!("CARGO_PKG_VERSION"), cross::commit_info())`